### PR TITLE
Update about.jsp & prototype profile

### PR DESCRIPTION
**Pull request by Sereezy**
It looks like a lot but all I did was change the font, the font size, the font color, and added different text with information about the team members. I accidentally kept pushing my trials and errors to github without realizing what I was doing until after. I made changes so that the profile page can only be seen when logged in and I changed the chat box so that authors names are links to their profile pages

### CODE REVIEW

**Review by DanielMcCrystal — APPROVED** [review]

**Comment by kyrabusser** [comment]
Nice work! :tada: Can you post a screenshot of what the new fonts and colors look like?

Let me know if you have any trouble with the "conflicting files" warning.

**Comment by Sereezy** [comment]
On Wednesday, May 16, 2018, 10:27:14 PM EDT, Kyra Busser <notifications@github.com> wrote:  
 
 
Nice work! 🎉 Can you post a screenshot of what the new fonts and colors look like?

Let me know if you have any trouble with the "conflicting files" warning.

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <nav>
+      <a id="navTitle" href="/">CodeU Chat App</a>
+      <a href="/conversations">Conversations</a>
+      <% if(request.getSession().getAttribute("user") != null){ %>
+        <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+      <% } else{ %>
+        <a href="/login">Login</a>
+      <% } %>
+      <a href="/about.jsp">About</a>
+      <<a href="/profile/<%=request.getSession().getAttribute("user")%>">Profile</a>
```
Comment (kyrabusser) [inline src/main/webapp/WEB-INF/view/profile.jsp]: Looks like there's an extra `<` here

File: src/main/java/codeu/controller/ActivityFeedServlet.java [hunk 2]
```diff
@@ -16,9 +16,9 @@ public void init() throws ServletException {
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response)
         throws IOException, ServletException {
-        
+
```
Comment (kyrabusser) [inline src/main/java/codeu/controller/ActivityFeedServlet.java:19]: Nit: You could undo the whitespace changes here and in ChatServlet.java--that way the history of these files (like you can see [here](https://github.com/Sereezy/CodeU-Spring-2018/commits/master/src/main/java/codeu/controller/ChatServlet.java)) won't show unrelated changes.

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ProfileServlet extends HttpServlet {
+	  @Override
```
Comment (kyrabusser) [inline src/main/java/codeu/controller/ProfileServlet.java:10]: Nit: the alignment here looks weird in GitHub to me? You could check if there's maybe a tab instead of spaces or something like that.

**Review by kyrabusser — APPROVED** [review]
Awesome! 🎉 A screenshot of the new profile prototype would also be great.

File: src/main/webapp/WEB-INF/view/activityfeed.jsp [hunk 4]
```diff
@@ -19,6 +19,11 @@
       <% } %>
       <a href="/about.jsp">About</a>
       <a href="/activityfeed">Activity Feed</a>
+      <% if (request.getSession().getAttribute("user") != null) { %>
+        <a href="/profile/<%=request.getSession().getAttribute("user")%>">Profile</a>
+      <% } else{ %>
```
Comment (kyrabusser) [inline src/main/webapp/WEB-INF/view/activityfeed.jsp:24]: No need to have an else here at all--you can just have the if part and do nothing otherwise (instead of putting an empty `a` tag that doesn't do anything).

File: src/main/webapp/WEB-INF/view/chat.jsp [hunk 5]
```diff
@@ -72,7 +77,8 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
         String author = UserStore.getInstance()
           .getUser(message.getAuthorId()).getName();
     %>
-      <li><strong><%= author %>:</strong> <%= message.getContent() %></li>
+    <%--  <li><strong><%= author %>:</strong> <%= message.getContent() %></li> --%>
+      <li><strong><a href="/profile/<%=request.getSession().getAttribute("author")%>"><%= author %>:</a></strong> <%= message.getContent() %></li>
```
Comment (kyrabusser) [inline src/main/webapp/WEB-INF/view/chat.jsp:81]: I think you can use the `author` variable in the link here, instead of getting an attribute from the session.

**Review by kyrabusser — APPROVED** [review]
Nice work! Let's go ahead and merge this into master today, then we can keep working on refining the feature in separate PRs. :smile:
